Assets/_Scripts/Player/VR/VRControl.cs

```diff
@@ -133,21 +153,20 @@ void Update()
     }
 
 #if !UNITY_WEBGL
-    private void SelectVRSystem(VRDevice device)
-    {
-        if (device == VRDevice.None) {
+    private void SelectVRSystem() {
+        deviceType = GetVRDevice();
+        if (deviceType == VRDevice.None) {
             XRSettings.enabled = false;
```

Comment:
my only comment is the question of if we should ever warn someone that we haven't found VR after looking.  I think that's probably just a separate future request tho.